qt/python/mantidqt/_common.sip

```diff
@@ -936,3 +936,34 @@ signals:
     void findingFiles();
     void filesFound();
 };
+
+// ---------------------------------
+// ImageInformationWidget
+// ---------------------------------
+class DisplayType {
+%TypeHeaderCode
+#include "MantidQtWidgets/Common/DisplayType.h"
+%End
+public:
+    DisplayType();
+	virtual void convertToDataCoord(const double xDisplayCoord, const double yDisplayCoord, double &xDataCoord, double &yDataCoord) = 0;
+};
+
+class ImageInfoWidget : public QTableWidget {
+%TypeHeaderCode
+#include "MantidQtWidgets/Common/ImageInfoWidget.h"
+%End
+public:
+    ImageInfoWidget(std::string &wsName, DisplayType *type, QWidget *parent = nullptr);
```

Comment:
I realised that changing to a workspace type makes this more complicated and there are no examples of it. Something like
```
    ImageInfoWidget(SIP_PYOBJECT ws, DisplayType *type, QWidget *parent = nullptr);
    %MethodCode
    try {
      auto extractor = Mantid::PythonInterface::ExtractWorkspace(boost::python::object(boost::python::handle<>(a0)));
      sipCpp = new sipImageInfoWidget(extractor(), a1, a2);
    } catch(...) {
      sipRaiseUnknownException();
      return NULL;
    }
    %End
```
should work.